README.md

```diff
@@ -79,15 +79,52 @@ more details) and implement the necessary logic.
 Then, the charm name needs to be added to `SUPPORTED_CHARMS` dictionary in
 `juju_verify/verifiers/__init__.py` *et voilà*, the charm is now supported.
 
-Don't forget to add unit and functional tests, and run:
+### Testing
+
+Don't forget to add unit and functional tests.
+
+#### Unittests
+
+You can run unittest with:
+
+```bash
+make unittest
+# or
+tox -e unit
+```
+However, we recommend running unittests at the same time as lint tests as follows:
 
 ```bash
-make test
+make tests
+# or
+tox
+```
+
+#### Functional tests
+
+Functional tests can be run using:
+
+```bash
+make functional
+# or
+tox -e func
 ```
 
 Functional tests require some applications to use a VIP. Please ensure the `OS_VIP00`
 environment variable is set to a suitable VIP address before running functional tests.
 
+We will also give useful examples of running tests during development.
```

Comment:
During development, different variations of all the functional tests may be run. Find some examples below: